monkey_island/cc/ui/src/components/pages/ReportPage.js

```diff
@@ -414,7 +417,7 @@ class ReportPageComponent extends AuthComponent {
           <ScannedServers data={this.state.report.glance.scanned}/>
         </div>
         <div>
-          <StolenPasswords data={this.state.report.glance.stolen_creds}/>
+          <StolenPasswords data={this.state.report.glance.stolen_creds, this.state.report.glance.ssh_keys}/>
```

Comment:
Pretty sure that's a bug and should be `[this.state.report.glance.stolen_creds, this.state.report.glance.ssh_keys]`

Reply:
It works, so I presume it's not a bug. Also, your mentioned solution does not build.